packages/utils/src/s3/S3Client.js

```diff
@@ -71,20 +69,19 @@ export class S3Client {
 
   /**
    * @public
-   * @param {string} filePath
+   * @param {string} fileName
+   * @param {*} base64EncodedFile
    * @returns
    */
-  async uploadFile(filePath) {
-    const fileName = path.basename(filePath);
-    const s3FilePath = `${getS3UploadFilePath()}${getUniqueFileName(fileName)}`;
+  async uploadFileFromBase64(fileName, base64EncodedFile) {
```

Comment:
Just a small thing on the naming of this, that I think this works with base64encoded or streams yeah?

Maybe just:
```
  /**
   * @public
   * @param {string} fileName
   * @param {*} fileStream | base64EncodedFile
   * @returns
   */
  async uploadFileFromBase64(fileName, file)
```